core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -517,7 +517,10 @@ angular.module('oppia').directive('questionsList', [
                 function(
                     $scope, $uibModalInstance, StateEditorService,
                     UndoRedoService) {
-                  var returnArray = [];
+                  var returnModalObject = {
+                    array: [],
+                    commitMessage: 'blank'
```

Comment:
No, please do *not* use commit messages of the form `'blank'`. That's sloppy and will cause confusion. Use the correct messages at all times.

Reply:
done